apps/accelerate/open_alpha_tensor/open_alpha_tensor/core/actors/stage.py

```diff
@@ -274,19 +306,14 @@ def monte_carlo_tree_search(
     state_hash = to_hash(extract_present_state(state))
     if state_hash in state_dict:
         with torch.no_grad():
-            N_s_a = state_dict[state_hash][3]
+            N_s_a = state_dict[state_hash][2]
```

Comment:
please keep the old code modification (the code in red should replace the code in green)